### PR TITLE
add isEii and isExtMethod to model

### PR DESCRIPTION
**Pull request by superyyrrzz**
@vwxyzh @qinezh @hellosnow @ansyral @chenkennt

### CODE REVIEW

File: src/Microsoft.DocAsCode.Build.ManagedReference/BuildOutputs/ApiBuildOutput.cs [hunk 1]
```diff
@@ -26,6 +26,14 @@ public class ApiBuildOutput
         [JsonProperty("id")]
         public string Id { get; set; }
 
+        [YamlMember(Alias = "isEii")]
+        [JsonProperty("isEii")]
+        public bool IsExplicitInterfaceImplementation { get; set; }
+
+        [YamlMember(Alias = "isExtMethod")]
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Build.ManagedReference/BuildOutputs/ApiBuildOutput.cs]: Ext -> Extension

**Comment by vwxyzh** [comment]
👍

**Comment by hellosnow** [comment]
:shipit:

**Comment by qinezh** [comment]
:dog:

**Pull request merged**